### ISSUE TITLE
'place' fields should autocomplete with fast/geographic and fast/corporate

### ISSUE THREAD

**Issue by hackmastera** [chemheritage/chf-sufia#136]
depends on #81

**Comment by jrochkind** [comment]
Why do we want to allow corporate names in the 'place' field?  Is FAST corporate vocab actually used for some geographic headings, or are we intentionally wanting to allow headings that aren't obviously geographic here?

Just trying to understand context.

**Comment by catlu** [comment]
If I'm remembering correctly, the latter--some curators wanted to be able to document the institution of origin as well, like Kodak or University of Chicago, that kinda thing.

**Comment by jrochkind** [comment]
Thanks @catlu! A corporate author like that, or whatever you want to call it, actually does make sense in a 'place' field, from your metadata evaluation?

**Comment by catlu** [comment]
Yep! Here's an example of Caltech being used: https://digital.chemheritage.org/catalog?_=1511798838400&f%5Bplace_facet_sim%5D%5B%5D=California+Institute+of+Technology&filter_public_domain=0&q=&search_field=all_fields

**Comment by jrochkind** [comment]
tech note: rather than replace our JS so it can pull from two sources, as anna suggests in #81, we could instead write a new source for the `qa` gem that pulls from both fast vocabs and combines them, somehow.
